openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -710,6 +706,47 @@ Cache-Control: no-store
 }
 ```
 
+# Nonce Endpoint {#nonce-endpoint}
+
+This endpoint allows a Client to acquire a fresh `c_nonce` value without the overhead of a full Credential Request. A Credential Issuer that requires `c_nonce` values to be incorporated into proofs in the Credential Request (see (#credential-request)) MUST offer a Nonce Endpoint.
```

Comment:
> without the overhead of a full Credential Request

I guess this refers to the "nonce by error" delivery mechanism, but this comes later in the text so it might be confusing to phrase it like this. Maybe we could drop this part?

```suggestion
This endpoint allows a Client to acquire a fresh `c_nonce` value. A Credential Issuer that requires `c_nonce` values to be incorporated into proofs in the Credential Request (see (#credential-request)) MUST offer a Nonce Endpoint.
```

Reply:
it was a nod to the folks who find the "nonce by error" delivery mechanism so deeply objectionable and I'd prefer to keep it until such time as #393 is acted on.